install_config/install/advanced_install.adoc

```diff
@@ -798,17 +798,54 @@ openshift_examples_modify_imagestreams=true
 |Set to `true` if pointing to a registry other than the default. Modifies the image stream location to the value of `*oreg_url*`.
 
 |`*openshift_docker_additional_registries*`
-|Specify the additional registry or registries.
-If the registry required to access the registry is other than `80` include the port number required.
+|Specify the additional registry or registries. If the registry required to access the registry is other than `80` include the port number required.
+
+|`*openshift_cockpit_deployer_prefix*`
+|Specify the URL and path to namespace where the registry-console image is located.
+
+|`*openshift_web_console_prefix*`
+|Specify the prefix for the web console images.
+
+|`*openshift_service_catalog_image_prefix*`
+|Specify the prefix for the service catalog component image.
+
+|`*ansible_service_broker_image_prefix*`
+|Specify the prefix for the ansible service broker component image.
+
+|`*template_service_broker_prefix*`
+|Specify the prefix for the template service broker component image.
+
+|`*openshift_crio_systemcontainer_image_override*`
+|A setting for if you are using CRI-O and if you are using an alternative CRI-O
+system container image from another registry.
 |===
 
 For example:
 ----
 oreg_url=example.com/openshift3/ose-${component}:${version}
 openshift_examples_modify_imagestreams=true
 openshift_docker_additional_registries=example.com:443
+openshift_cockpit_deployer_prefix='registry.example.com/openshift3/'
```

Comment:
I do not know how best to present this image. The names are completely different for origin vs enterprise, and from other similar images.

[Origin](https://github.com/openshift/openshift-ansible/blob/18961e08a744981f0a7726f653edf0c981ff67e6/roles/openshift_hosted_templates/files/v3.9/origin/registry-console.yaml#L103): default is `cockpit/` and to use an alternate registry, should be something like `registry.example.com/cockpit/` unless of course they renamed it to something sane. Note the rest of the image name defaults to `kubernetes` so `cockpit/kubernetes:latest`.

[Enterprise](https://github.com/openshift/openshift-ansible/blob/18961e08a744981f0a7726f653edf0c981ff67e6/roles/openshift_hosted_templates/files/v3.9/enterprise/registry-console.yaml#L103): default is more like usual, so `registry.example.com/openshift3/` would work for alternate registry.

Note there are variables in the template for specifying the rest of the image, and ansible variables that feed into them. I don't know that you want to mention them here. If you get into listing every single variable that can impact image names this will be quite a list.

Reply:
Hmm right. Thanks @sosiouxme . I've included what you've mentioned here in the table above when it describes the parameter.